src/app-layout/visual-refresh/trigger-button.tsx

```diff
@@ -35,11 +36,12 @@ function TriggerButton(
         className
       )}
       onClick={onClick}
-      type="button"
       ref={ref as React.Ref<HTMLButtonElement>}
+      type="button"
       {...focusVisible}
```

Comment:
There might be merge conflicts because I changed this line #934

If everything resolves correctly after rebase, I can approve

Reply:
All set!